components/frontend/src/widgets/TableRowWithDetails.js

```diff
@@ -1,27 +1,23 @@
+import "./TableRowWithDetails.css"
+
 import { bool, func, object } from "prop-types"
 
-import { Icon, Table } from "../semantic_ui_react_wrappers"
+import { Button, Icon, Table } from "../semantic_ui_react_wrappers"
 import { childrenPropType } from "../sharedPropTypes"
 
 export function TableRowWithDetails(props) {
     const { children, details, expanded, onExpand, style, ...otherProps } = props
     return (
         <>
             <Table.Row {...otherProps}>
-                <Table.Cell
-                    aria-label="Expand/collapse"
-                    collapsing
-                    onClick={() => onExpand(!expanded)}
-                    onKeyPress={(event) => {
-                        event.preventDefault()
-                        onExpand(!expanded)
-                    }}
-                    tabIndex="0"
-                    textAlign="center"
-                    style={style}
-                    role="button"
-                >
-                    <Icon size="large" name={expanded ? "caret down" : "caret right"} />
+                <Table.Cell collapsing textAlign="center" style={style}>
+                    <Button
```

Comment:
Is there no functional test that requires altering due to this change?

Reply:
No, because the Table.Cell was rendered as a button before (see https://react.semantic-ui.com/augmentation/) so all the tests keep working.